pages/index.vue

```
@@ -2,17 +2,29 @@
-        clearable
-        size="large"
-        :prefix-icon="ElIconSearch"
-        class="search-input"
-      />
+      <div class="search-container">
+        <ElButton
+          v-if="hasGeolocation"
+          @click="toggleLocationFilter"
+          size="large"
+          type="default"
+          :class="['location-button', { 'location-active': locationPermissionGranted }]"
```

ESLint

[error] Replace `'location-button',·{·'location-active':·locationPermissionGranted·}` with `⏎············'location-button',⏎············{·'location-active':·locationPermissionGranted·},⏎··········`

(prettier/prettier)